src/docs/background/13-how-payment-intents-work/index.md

```diff
@@ -0,0 +1,187 @@
+3DS, in addition to supporting payment authentication in your client app, you
+may need to update your [Stripe Radar
+rules](https://stripe.com/docs/payments/3d-secure#three-ds-radar).
+
+### Handling Strong Customer Authentication
+
+[Strong Customer
+Authentication](https://stripe.com/en-fi/payments/strong-customer-authentication)
+is a potential step enforced by governmental regulation. Not every PaymentIntent
+will require customer authentication. For instance, authentication may not be
+required for:
+
+- transactions out of scope of SCA
+- e.g. when card issuing bank is outside of EEA
+- merchant initiated transactions
+- transactions that fall under an SCA exemption
+- low value or low risk transactions
+- recurring payments for fixed amount
+- other
+
+This means that Flex implementation of PaymentIntents supports payment flows
+that require authentication and those that do not. When implementing the
+PaymentIntent flow in the client you need to be prepared for handling both
+cases - payments requiring SCA and payments that do not. It might be impossible
+to know in advance whether the payment will require authentication, unless the
+marketplace and all its customers are outside of EEA.
+
+The recommended way of implementing support for SCA is to use [Stripe
+Elements](https://stripe.com/docs/payments/payment-intents/quickstart) that can
+provide you with ready modals for handling e.g. 3D Secure Card Payments. The
+next chapter will provide high level instructions on how to do this in the
+client.
+
+### Implementing the automatic PaymentIntent flow
+
+[Stripe Elements](https://stripe.com/docs/payments/payment-intents/quickstart)
+provides ready tools and a reference for implementing the automatic
+PaymentIntent flow. It's the recommended way to support PaymentIntents in the
+client and handle the authentication steps.
+
+For implementing the PaymentIntent flow, you need to follow the steps in
+[Automatic confirmation
+quickstart](https://stripe.com/docs/payments/payment-intents/quickstart#automatic-confirmation-flow).
+With Flex, the first step differs from Stripe instructions, since that is
+handled through the transaction engine with the `stripe-create-payment-intent`
+action. To complete the first step, you initiate (or transition) the transaction
+so that a PaymentIntent is created.
+
+For handling the next steps, you need to [pass in a client
+secret](https://stripe.com/docs/payments/payment-intents/quickstart#passing-to-client).
+
+The client secret is exposed in the transaction's protecedData map under a
+well-known key `stripePaymentIntents` after the PaymentIntent has been created.
+The value is a map in the form of:
+
+```json
+{
+ "default": {
+    "stripePaymentIntentId": "pi_1EXSEzLSea1GQQ9x5PnNTeuS",
+    "stripePaymentIntentClientSecret": "pi_1EXSEzLSea1GQQ9x5PnNTeuS_secret_Qau2uE5J5L6baPs8eLPMa2Swb"
+  }
+}
+```
+
+This data is only exposed to the customer in the transaction. The provider can
```

Comment:
@sktoiva I think we should directly tell where the client secret is:
`transaction.attributes.protectedData.stripePaymentIntents.default.stripePaymentIntentClientSecret`
Or alternatively have a link to API reference, where the secret is mentioned.
(I'm not sure if it is mentioned there...)

Reply:
"well-known key" is a bit of overstatement :)